tutorials/unit_tutorial.py

```diff
@@ -0,0 +1,128 @@
+print(area)
+
+#########################################################################
+# That's great, now we have an area, but it is not in a very useful unit still.
+# Units can be converted using the `to()` method. While you won't see square meters in
+# the units list, we can parse complex/compound units as strings:
+
+print(area.to('m^2'))
+
+#########################################################################
+# Temperature
+# -----------
+# Temperature units are actually relatively tricky (more like absolutely tricky as
+# you'll see). Temperature is a non-multiplicative unit - they are in a system
+# with a reference point. That means that not only is there a scaling factor, but
+# also an offset. This makes the math and unit book-keeping a little more complex.
+# Imagine adding 10 degrees Celsius to 100 degrees Celsius. Is the answer 110
+# degrees Celsius or 383.15 degrees Celsius (283.15 K + 272.15 K)? That's why
+# there are delta degrees units in the unit registry for offset units. For more
+# examples and explanation you can watch MetPy Monday #13:
+# https://www.youtube.com/watch?v=iveJCqxe3Z4.
+#
+# Let's take a look at how this works and fails:
+#
+# We would expect this to fail because we cannot add two offset units (and it does
+# fail as an "Ambiguous operation with offset unit").
+#
+# `10 * units.degC + 5 * units.degC`
+#
+# On the other hand, we can subtract two offset
+# quantities and get a delta:
+
+print(10 * units.degC - 5 * units.degC)
+
+#########################################################################
+# We can add a delta to an offset unit as well:
+
+print(25 * units.degC + 5 * units.delta_degF)
+
+#########################################################################
+# Absolute temperature scales like Kelvin and Rankine do not have an offset
+# and therefore can be used in addition/subtraction without the need for a
+# delta version of the unit.
+
+print(273 * units.kelvin + 10 * units.kelvin)
+
+#########################################################################
+
+print(273 * units.kelvin - 10 * units.kelvin)
+
+#########################################################################
+
+#########################################################################
+# Compound Units
+# --------------
+# We can create compound units for things like speed by parsing a string of
+# units. Abbreviations or full unit names are acceptable.
+
+u = np.random.randint(0, 15, 10) * units('m/s')
+v = np.random.randint(0, 15, 10) * units('meters/second')
+
+print(u)
+print(v)
+
+#########################################################################
+# Common Mistakes
+# ---------------
+# There are a few common mistakes the new users often make. Be sure to check
+# these when you're having issues
+#
+# * Pressure units are `mbar` or `hPa` for common atmospheric measurements. The
+#   unit `mb` is actually millibarns.
+# * When using masked arrays, units must be multiplied on the left side. This
```

Comment:
Would be good to indicate how it fails when you multiply on the right (which of course actually works sometimes).

Reply:
Done